script/test.sh

```diff
@@ -61,6 +61,29 @@ test_check () {
   fi
 }
 
+test_PR38 () { # PR38: Feature error handling by git rebase --abort
+   prepare_env
+   git_init
+   git_pre_commit
+   git revert HEAD~2
+   echo file-11 >> file-11
+   git add file-11
+   git commit -m "Add file-11"
+
+   set +e
+   REFLOG_HASH_1=$(git log --oneline --format=%h|head -n1)
+   ./${ExecComamnd}  -n 2..5 -f -d
+   REFLOG_HASH_2=$(git log --oneline --format=%h|head -n 1)
+   set -e
+   if [ "${REFLOG_HASH_1}" == "${REFLOG_HASH_2}" ]; then
+     echo "[passed] RUN ./${ExecComamnd}  -n 2..5 -f -d"
```

Comment:
please please write to text summary

> >> ./../test-$$-result

L81 too.

Reply:
Added it.